dev-packages/e2e-tests/test-applications/ember-classic/tests/errors.test.ts

```diff
@@ -3,7 +3,7 @@ import { waitForError, waitForTransaction } from '@sentry-internal/test-utils';
 
 test('sends an error', async ({ page }) => {
   const errorPromise = waitForError('ember-classic', async errorEvent => {
-    return !errorEvent.type;
+    return errorEvent.type !== 'transaction';
```

Comment:
m: I don't think this change is correct because event.type could be a lot more than `'transaction'` or  `undefined`. However, it probably doesn't matter because `waitForError` already checks for `type === undefined` before even invoking the callback :)